### DISCUSSION TITLE
⁉️ Copilot stopped working 2 months ago ⁉️

### DISCUSSION THREAD

**Discussion by jbruxelle** [community/community#11084] [Copilot Conversations]
Hi, 

I have been facing issues with Github Copilot for 2 months now... Explanation :

- I use `VSCode` ;
- I am under `WSL2 in remote connection` (Copilot installed in WLS2);
- The Copilot icon (with ❗) shows the following message on hover: `"Copilot is encountering temporary issues"` ;
- The output logs for Copilot when I write code are the following : 
 ```
[INFO] [ghostText] [2022-02-06T08:44:39.304Z] Breaking, no choices
[INFO] [default] [2022-02-06T08:44:39.461Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-aatest
```

I tried uninstalling, reinstalling, launching and relaunching VSCode... and it almost works... but after 1 or 2 code suggestions, Copilot stops working.

Any suggestion, solution to this problem ?

**Comment by davecheney** [comment]
Could you post a screenshot, or better, a recording, of what you are experiencing?

**Comment by davecheney** [comment]
Does copilot work correctly when not using a remote WSL2 connection?

**Comment by jbruxelle** [comment]
So, as you can see, it works on Windows (I tested it thanks to your question) but on WLS2 in only gives one suggestion and then faces issues... it seems that the engine is not able to fetch completions or something like that.

So the problem can be related to my personal config but I was wondering if someone else had already encountered the same issues.
I know from someone who I work with that it can work using WSL2 remote connection on VSCode but I can figure out what differs...

https://user-images.githubusercontent.com/36154424/152676408-ec21d337-6ac1-4f70-b981-5f6a03a5008b.mp4

I will search for issues on WSL2 not able to reach windows internet connection.

**Reply by davecheney** [reply]
Thank you for the recording. Sadly it appears corrupt when I try to play it back.

**Reply by jbruxelle** [reply]
You mean you can't read it ? I just tried reading it from my phone using the Github app and it works fine. Yet there is a green background, maybe due to the fact I recorded it on my 34" screen which... is not a regular format (ultra wide).

**Comment by jbruxelle** [comment]
Similar to #7153 (not answered) and #6785
